test/test_nn.py

```diff
@@ -7019,6 +7019,17 @@ def test_cross_entropy_loss_zero_div(self):
         input_2 = torch.rand([5, 0], dtype=torch.float32)
         torch.nn.CrossEntropyLoss()(input_1, input_2)
 
+    @parametrize_test("reduction", ("none", "mean", "sum"))
+    def test_cross_entropy_dim_parameter(self, device, reduction):
```

Comment:
This test should be moved underneath the `TestNNDeviceType` class so it is automatically instantiated across devices. [Here](https://github.com/pytorch/pytorch/blob/5210d7c4233f115d57a09cfe7d96c2a32a057f7c/test/test_nn.py#L11575) is a good place to put it

Reply:
I've moved the test